libs/oidc/ngx/README.md

```diff
@@ -1,5 +1,46 @@
 # OIDC Ngx Common Routes
 
+# tamu-gisc/ngx-authentication-guard
+
+## Motivation
+
+This was created for use within `gisday-angular` but I felt it could be useful for any client Angular project. This guard will lock resources behind a user login.
```

Comment:
What does this do differently from the guards provided by the `angular-auth-oidc-client`?